packages/msw-addon/src/loader.ts

```diff
@@ -0,0 +1,18 @@
+import type { Context } from './decorator'
+import { applyRequestHandlers } from './applyRequestHandlers'
+
+export const mswLoader = async (context: Context) => {
+  applyRequestHandlers(context.parameters.msw)
```

Comment:
@yannbf, I think you will love how simple these functions become! I've also moved the singleton `api` access to `applyRequestHandlers`, which means we keep a single point of entry when we decide to refactor something.